client/app/telecom/telephony/alias/portability/order/telecom-telephony-alias-portability-order.html

```diff
@@ -0,0 +1,878 @@
+    </header>
+
+    <div class="telecom-section-content">
+        <div class="row" >
+
+            <!-- ORDER SUCCESS -->
+            <div class="col-md-6 widget-presentation"
+                 data-ng-if="PortabilityOrderCtrl.order.success">
+                <header class="widget-presentation-header">
+                    <h2 class="widget-presentation-title"
+                        data-translate="telephony_alias_portability_order_order"></h2>
+                </header>
+                <div class="alert alert-success" role="alert">
+                    <i class="ovh-font ovh-font-filled-check" aria-hidden="true"></i>
+                    <p data-translate="telephony_alias_portability_order_success"></p>
+                </div>
+                <p>
+                    <!-- BC LINK -->
+                    <a target="_blank"
+                        class="btn btn-primary"
+                        rel="noopener"
+                        data-ng-if="PortabilityOrderCtrl.order.url"
+                        data-ng-href="{{:: PortabilityOrderCtrl.order.url }}">
+                        <span data-translate="telephony_alias_portability_order_url"></span>
+                        <i class="fa fa-external-link" aria-hidden="true"></i>
+                    </a>
+                </p>
+            </div>
+
+            <div class="col-md-6 widget-presentation"
+                 data-ng-if="!PortabilityOrderCtrl.order.success">
+                <header class="widget-presentation-header">
+                    <h2 class="widget-presentation-title"
+                        data-translate="telephony_alias_portability_order_order"></h2>
+                </header>
+
+                <small class="opacity5"
+                      data-translate="telephony_alias_portability_order_step"
+                      data-translate-values="{
+                          current: PortabilityOrderCtrl.stepsList.indexOf(PortabilityOrderCtrl.step) + 1,
+                          total: PortabilityOrderCtrl.stepsList.length
+                      }">
+                </small>
+                <ol class="breadcrumb">
+                    <li data-ng-class="{ opacity5: PortabilityOrderCtrl.step !== 'number' }"
+                        data-translate="telephony_alias_portability_order_step_1"></li>
+                    <li data-ng-class="{ opacity5: PortabilityOrderCtrl.step !== 'contact' }"
+                        data-translate="telephony_alias_portability_order_step_2"></li>
+                    <li data-ng-class="{ opacity5: PortabilityOrderCtrl.step !== 'config' }"
+                        data-translate="telephony_alias_portability_order_step_3"></li>
+                    <li data-ng-class="{ opacity5: PortabilityOrderCtrl.step !== 'summary' }"
+                        data-translate="telephony_alias_portability_order_step_4"></li>
+                </ol>
+
+                <div class="col-xs-12">
```

Comment:
bootlint:  E014  Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s